osi_lane.proto

```diff
@@ -375,6 +375,11 @@ message Lane
         // This subtype specifies a lane more concretely.
         //
         optional Subtype subtype = 12;
+        
+        // The list of surfacelines of the lanes.
+        // 
+        // This surfacelines decribe the height or road shape profile along the lane.
```

Comment:
I think there should be an explanation that currently the field is undefined for intersection as you write in the explanation of the PR. We should also discuss non-driving lanes. Suggestion:
```suggestion
        // This surfacelines decribe the height or road shape profile along the lane.
        //
        // \note The \c #left_lane_boundary_id is currently undefined for \c #type =
        // \c #TYPE_INTERSECTION .
        //
        // \note OSI uses singular instead of plural for repeated field names.
        //
```